From e5e0c66d632e9a81cce0698d8b61a71deaad4b0b Mon Sep 17 00:00:00 2001
From: libjared <libjared@users.noreply.github.com>
Date: Tue, 7 Jun 2022 22:35:23 -0500
Subject: [PATCH] Remove upper bound on aeson

---
 ghcjs-base.cabal | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ghcjs-base.cabal b/ghcjs-base.cabal
index 7012e61..f18855f 100644
--- a/ghcjs-base.cabal
+++ b/ghcjs-base.cabal
@@ -145,7 +145,7 @@ library
                    binary               >= 0.8  && < 0.11,
                    bytestring           >= 0.10 && < 0.11,
                    text                 >= 1.1  && < 1.3,
-                   aeson                >= 0.8  && < 1.6,
+                   aeson                >= 0.8,
                    scientific           >= 0.3  && < 0.4,
                    vector               >= 0.10 && < 0.13,
                    containers           >= 0.5  && < 0.7,
-- 
2.35.1

